packages/ember-htmlbars/tests/integration/mutable_binding_test.js

```diff
@@ -111,7 +111,7 @@ QUnit.test('using a string value through middle tier does not trigger assertion'
 
   runAppend(view);
 
-  assert.strictEqual(bottom.attrs.stuff.value, 'foo', "precond - the data propagated");
+  assert.strictEqual(bottom.attrs.stuff, 'foo', "precond - the data propagated");
```

Comment:
Not sure I follow.  the `bottom.attrs.stuff.value` was not asserting that it was a stream, but that it was a `MUTABLE_CELL`.  I believe that in this case, the assertion was correct.

The invoking site was `{{bottom-mut stuff=attrs.value}}`, and with "auto-mut" mode that should result in the same thing as `{{bottom-mut stuff=(mut attrs.value)}}` which would be a `MUTABLE_CELL`.